Traceback (most recent call last):
  File "D:/Users/Admin/PycharmProjects/PyCharm_Projects/Deep_learning/cat_dog/model_training02.py", line 103, in <module>
    model = mnist.MNIST(num_classes = 1)
TypeError: __init__() got an unexpected keyword argument 'num_classes'
Traceback (most recent call last):
  File "D:/Users/Admin/PycharmProjects/PyCharm_Projects/Deep_learning/cat_dog/model_training02.py", line 103, in <module>
    model = mnist.MNIST(num_classes = 1)
  File "D:\Users\Admin\PycharmProjects\PyCharm_Projects\Deep_learning\cat_dog\mnist.py", line 13, in __init__
TypeError: __init__() got an unexpected keyword argument 'filter_size'
